src/components/PrivateRoute/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
```

Comment:
Not necessary as Next.js automatically imports react

Reply:
Nice